### PR TITLE
buffs IEDs [ready]

### PR DESCRIPTION
**Pull request by ghost**
## zamolxius
:cl:
tweak: made IEDs actually explode
/:cl:

## About The Pull Request
buffs IEDs to explode like they did before tg nerfed them
## Why It's Good For The Game
pre rebase port, i guess? here's a pic

Y1MxEwr

### CODE REVIEW

**Comment by mikolaj007** [comment]
if the gif is the buffed version +1
what the fuck happened to IED's

**Comment by ghost** [comment]
yes gif is buffed,  tg made them just spew flames around,  not even a proper ied smh.

File: hippiestation/code/game/objects/items/grenades/ghettobomb.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/obj/item/grenade/iedcasing/prime() //buffs IEDs
+	update_mob()
+	explosion(src.loc,-1,0,2, flame_range = 2)
```
Comment (yoyobatty) [inline hippiestation/code/game/objects/items/grenades/ghettobomb.dm:3]: old code was: 	`explosion(src.loc,-1,-1,2, flame_range = 4)	// small explosion, plus a very large fireball.`
So added a heavy range for direct impact memes, and reduced the flame range by 2. I like it.

**Comment by cris5311** [comment]
+1
Finally they arent utterly useless. Id buff the flame range to 3

File: hippiestation/code/game/objects/items/grenades/ghettobomb.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/game/objects/items/grenades/ghettobomb.dm:3]: its good for bombing APCs and the like bro.  better than flame shart

**Comment by yoyobatty** [comment]
It seems like you changing the heavy impact from -1 to 0 didn't do anything. I created 2 explosions, one was the old, one was the new on 2 humans and the damage difference was negligible. The flame range reduction will probably just make them less laggy, that's it.

**Comment by ghost** [comment]
hmm,  it knocked me out when I tested it

**Comment by ghost** [comment]
i tweak it for bigger explosion with no to-space destruction?

**Comment by yoyobatty** [comment]
Actually, hold the phone. I was wearing armor. Damage on old was 60, damage on new was 75, unclothed mobs fyi. Seems decent to me.

**Review by yoyobatty — APPROVED** [review]

**Comment by Majick1** [comment]
+1, not much to add on this one

**Comment by ghost** [comment]
I assume that's a +1 from @yoyobatty so I'm going to merge this nice buff. I remember once blowing up a locker of like 30 of these right next to me and only going into orange.

**Pull request merged**